src/test/java/me/moodcat/backend/rooms/RoomBackendTest.java

```diff
@@ -167,30 +171,46 @@ public void playSongCreatesQueueWhenRepeating() {
         instance.playNext();
         instance.merge();
 
-        assertNotEquals(room.getPlayHistory(), room.getPlayQueue());
+        assertFalse(room.getPlayQueue().isEmpty());
```

Comment:
This makes a new purpose for the test.
To test if the queue is different from the history, is something else than checking if the queue is not empty.
Is this intentional?